music21/abcFormat/testFiles.py

```diff
@@ -641,26 +637,20 @@
 
 '''
 
-
-# ------------------------------------------------------------------------------
-
 ALL = [fyrareprisarn, mysteryReel, fullRiggedShip, aleIsDear, kitchGirl,
-        williamAndNancy, morrisonsJig, hectorTheHero, kingOfTheFairies,
-        sicutRosa, theAleWifesDaughter, theBeggerBoy, theBattleOfTheSnaBas,
-        draughtOfAle,
-        valentineJigg,
-        testPrimitive, testPrimitivePolyphonic, testPrimitiveTuplet
+       williamAndNancy, morrisonsJig, hectorTheHero, kingOfTheFairies,
+       sicutRosa, theAleWifesDaughter, theBeggerBoy, theBattleOfTheSnaBas,
+       draughtOfAle,
+       valentineJigg,
+       testPrimitive, testPrimitivePolyphonic, testPrimitiveTuplet
        ]
 
 
 def get(contentRequest):
-    '''
-    Get test material by type of content
-    '''
+    """Get test material by type of content"""
```

Comment:
this is incorrect -- sorry.  Music21 house style is single quoted docstrings with a new line before and after.